marketplacetests/payment/app.py

```diff
@@ -6,8 +6,10 @@
 from marionette.by import By
 from marionette.wait import Wait
 
+from marketplacetests.marketplace_base import MarketplaceBase
 
-class Payment(Base):
+
+class Payment(MarketplaceBase):
```

Comment:
I find this refactor a little confusing. Are 'Marketplace' and 'Payment' separate apps? If I find time a little later I'll try to understand the architecture better, but it would seem to be that if 'Payment' is within the 'Marketplace' app, then it should extend the Marketplace app object. I'm not keen on having a base app object shared by two distinct apps.